string object_class
float32 confidence
Point bottom_left
Point top_right

